main.go

```diff
@@ -47,6 +47,8 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var installation = os.Getenv("INSTALLATION")
```

Comment:
why an env and not a flag like other fields?

Reply:
I'm an idiot, I haven't pushed the change 🤦🏻